modules/button/react/lib/IconButton.tsx

```diff
@@ -52,7 +52,7 @@ const IconButton: ButtonOrAnchorComponent<
 > = ({
   theme = useTheme(),
   variant = IconButtonVariant.Circle,
-  size = 'medium',
+  size = IconButtonVariant.ToolbarSquare ? 'small' : 'medium',
```

Comment:
Thoughts on this approach? I wanted to default to small always, but should we leave it open and have the consumer always set the size?